index.html

```diff
@@ -1,49 +1,296 @@
+            <li class="tag">javascript</li>
+          </ul>
+          <div class="actionBtn">
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="card" id="card3">
+        <div class="portfolioPic portfolioPic3"></div>
+        <div class="workDescription">
+          <div class="project">
+            <h3 class="projectTitle">Tonic</h3>
+            <ul class="clientDetail">
+              <li class="clientName">CANOPY</li>
+              <li><img src="./Assets/images/dot.svg" alt="dot" /></li>
+              <li>Back End Dev</li>
+              <li><img src="./Assets/images/dot.svg" alt="dot" /></li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="projectDesc">
+            <span
+              >A daily selection of privately personalized reads; no accounts or
+              sign-ups required.</span
+            >
+          </div>
+          <ul class="projectTags">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+          </ul>
+          <div class="actionBtn">
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="card" id="card4">
+        <div class="portfolioPic portfolioPic4"></div>
+        <div class="workDescription">
+          <div class="project">
+            <h3 class="projectTitle">
+              Multi-Post <br />
+              Stories
+            </h3>
+            <ul class="clientDetail">
+              <li class="clientName">CANOPY</li>
+              <li><img src="./Assets/images/dot.svg" alt="dot" /></li>
+              <li>Back End Dev</li>
+              <li><img src="./Assets/images/dot.svg" alt="dot" /></li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="projectDesc">
+            <span
+              >A daily selection of privately personalized reads; no accounts or
+              sign-ups required.</span
+            >
+          </div>
+          <ul class="projectTags">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+          </ul>
+          <div class="actionBtn">
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="aboutMyself">
+      <div class="aboutMe">
+        <h2 class="aboutTitle">
+          About <br />
+          Myself
+        </h2>
+        <p class="aboutDesc">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="aboutSM">
+          <p class="titleSM">LET’S CONNECT</p>
```

Comment:
- [ ] This text "LET'S CONNECT" should use a header `hx` tag instead of a `p` tag. This is important because it serves as a heading for the social media icons, hence, it is semantically correct to use the `hx` tag except that it has to be any other header tag and not `h1`. Kindly fix this

Reply:
Hi @bobb-Rob ,

I have made this "p" tag to "h4" tag.